src/app/proposals/shared/supportedABIs.ts

```diff
@@ -1,32 +1,18 @@
 import { DAOTreasuryAbi } from '@/lib/abis/DAOTreasuryAbi'
 import { RIFTokenAbi } from '@/lib/abis/RIFTokenAbi'
-import { SimplifiedRewardDistributorAbi } from '@/lib/abis/SimplifiedRewardDistributorAbi'
-import { BuilderRegistryAbi } from '@/lib/abis/v2/BuilderRegistryAbi'
+import { BuilderRegistryAbi } from '@/lib/abis/collective-rewards/BuilderRegistryAbi'
 import { HTMLProps } from 'react'
 import { AbiFunction, AbiParameterToPrimitiveType } from 'viem'
 
-export const abiNames = [
-  'DAOTreasuryAbi',
-  'RIFTokenAbi',
-  'BuilderRegistryAbi',
-  // CR MVP: To keep compatibility with the MVP logs
-  'SimplifiedRewardDistributorAbi',
-] as const
-export const abis = [
-  DAOTreasuryAbi,
-  RIFTokenAbi,
-  BuilderRegistryAbi,
-  // CR MVP: To keep compatibility with the MVP logs
-  SimplifiedRewardDistributorAbi,
-] as const
+export const abiNames = ['DAOTreasuryAbi', 'RIFTokenAbi', 'BuilderRegistryAbi'] as const
+
+export const abis = [DAOTreasuryAbi, RIFTokenAbi, BuilderRegistryAbi] as const
 
 export const supportedProposalActions = [
   'withdraw',
   'withdrawERC20',
   'communityApproveBuilder',
   'dewhitelistBuilder',
-  // CR MVP: To keep compatibility with the MVP logs
-  'removeWhitelistedBuilder',
```

Comment:
Here I think that we may do it since there is no existing proposal, but I would keep it to provide compatibility.